test/lib/s3_test.exs

```diff
@@ -525,6 +525,68 @@ defmodule ExAws.S3Test do
     assert expected == S3.delete_object_tagging(bucket, object)
   end
 
+  test "#presigned_post simple policy" do
+    bucket = "my-bucket"
+    key = "test.txt"
+
+    post_data = S3.presigned_post(ExAws.Config.new(:s3), bucket, key)
+
+    assert URI.parse(post_data.url).scheme != nil
+    assert policy = post_data.fields["Policy"]
+
+    assert {:ok, json} = Base.decode64(policy)
+    assert {:ok, policy} = config.json_codec.decode(json)
+
```

Comment:
As noted above, this should be `config()` since it's a function call not a variable. The compiler has picked it up in this case.